gateway-service-impl/src/test/java/org/hypertrace/gateway/service/baseline/BaselineServiceQueryParserTest.java

```diff
@@ -75,6 +75,8 @@ public void testQueryResponse() {
                   {"entity-id-1", "1608525840000", "16.0"},
                   {"entity-id-1", "1608525540000", "17.0"}
                 }));
+    long endTimeInMillis = System.currentTimeMillis();
```

Comment:
Generally a red flag to use a variable value like this in a test. It looks like its only being given as an input (which is why it's not an issue), but that indicates we're not verifying how it's being used.

Reply:
Yes, we are not verifying startTime and endTime in test cases. Added basic tests, more tests can be added in next iteration.

Reply:
StartTime and endTime verified separately in testcases.

Reply:
Updated this to use startTimes like other tests.